src/patternfly/components/TextInputGroup/text-input-group.scss

```
@@ -49,8 +49,9 @@
 
   // Icon
   --#{$text-input-group}__icon--FontSize: var(--pf-t--global--icon--size--font--body--default);
-  --#{$text-input-group}__icon--InsetInlineStart: var(--pf-t--global--spacer--sm);
-  --#{$text-input-group}__icon--m-status--InsetInlineEnd: var(--pf-t--global--spacer--sm);
+  --#{$text-input-group}__icon--InsetInlineStart: var(--pf-t--global--spacer--control--horizontal--default);
+  --#{$text-input-group}__icon--m-status--InsetInlineEnd: var(--pf-t--global--spacer--control--horizontal--default);
+  --#{$text-input-group}__c-utilities__icon--m-status--InsetInlineEnd: var(--pf-t--global--spacer--control--horizontal--compact);
```

Just a couple of nits on this
* The syntax for this var is flexible since it's a pretty uncommon pattern. Though typically we reserve the `c-[whatever]` naming when we reference another component (eg `c-button`). We'll use `--[component/BEM element]` (eg `--c-button` or `--utilities`) as a general identifier (not necessarily a child or direct child, just in there somewhere) so I wonder if the suggestion below is better?
* Also not sure if `--pf-t--global--spacer--control--horizontal--compact)` is the right spacer. Reason being, if a person themed their compact control padding, I wouldn't expect this spacing to update? Since it doesn't necessarily need to conform to an outer button/control padding to match other controls, I think we could just use the `--spacer--sm` token here. WDYT?

```suggestion
  --#{$text-input-group}--utilities--icon--m-status--InsetInlineEnd: var(--pf-t--global--spacer--control--horizontal--compact);
```